streams/src/test/java/org/apache/kafka/streams/kstream/internals/TimeWindowedKStreamImplTest.java

```diff
@@ -244,6 +246,45 @@ public void shouldMaterializeAggregated() {
         }
     }
 
+    @Test
```

Comment:
This is a sample usage to materialize the windowed KTable. The syntax is still not ideal here, but could work.